blazor/datagrid/cell.md

```diff
@@ -347,4 +347,16 @@ The following screenshot represents a clip mode in DataGrid
 
 ![Clip Mode in Blazor DataGrid](./images/blazor-datagrid-clip-mode.png)
 
-N> You can refer to the [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
\ No newline at end of file
+N> You can refer to the [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
+
+## To display the tooltip when using DataGrid inside the Dialog Component
+
+Adding the below style in the application will display the tooltip.
```

Comment:
explain the cause of the issue and why this solution is being suggested. when it will be useful